docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM continuumio/miniconda3
+
+# update the conda packages
+RUN conda update -y conda pip
+
+# install development tools
+RUN apt-get update -y && apt-get install -y \
+    build-essential \
+    curl \
+    unzip
+
+# create directory for downloaded files
+RUN mkdir /src
+WORKDIR /src
+
+# create a new conda environment
+RUN conda create -y -n esmvaltool python=3
```

Comment:
Why create an `esmvaltool` environment when you install in the root environment below? I think this line is not needed.

Reply:
DONE